making-aeternity-wallet-transactions-with-aepp-sdk-js.md

```diff
@@ -0,0 +1,134 @@
+ 
+ ### New KeyPair Creation Without Existing Public Key
+```javascript
+ function generateKeyPairWithoutSecretKey(){
+    let { secretKey, publicKey } = Ae.Crypto.generateKeyPair(true);
+    console.log(secretKey);
+    console.log(publicKey);
+    let translatedPublicKey=Ae.Crypto.aeEncodeKey(publicKey);
+    let translatedSecretKey=buffer.Buffer.from(secretKey).toString('hex');
+    return {publicKey:translatedPublicKey,privateKey:translatedSecretKey};
+}
+```
+ The function above simply creates a brand new KeyPair, we do this by accessing the `generateKeyPair` method of the  `Crypto` module of the aepp-sdk-js. Then we repeat the same steps we did earlier to get the string version of our secret key and public key.
+ Voilà, we now know how to create a wallet on the Aeternity Blockchain.  Lets Quickly get into checking the bank account of this wallet.
+ 
+## Checking The Balance Of A Wallet
+To check a an aeternity wallet balance we need an instance of the sdk, so lets start by creating an sdk instance
+ ```javascript
+async function getSdkInstance(secretKey,publicKey){
+    const NODE_URL = 'https://sdk-testnet.aepps.com';
+    const ACCOUNT = Ae.MemoryAccount({ keypair: { secretKey: secretKey, publicKey: publicKey } });
+    const nodeInstance = await Ae.Node({ url: NODE_URL })
+    const sdkInstance = await Ae.Universal({
+       compilerUrl: 'https://compiler.aepps.com',
+       nodes: [ { name: 'test-net', instance: nodeInstance } ],
+       accounts: [ ACCOUNT ]
+    });
+    return sdkInstance;
+}
+```
+To create an SDK instance, we need to specify a Node Url `NODE_URL` which states if to connect to the mainnet or the testnet, Connecting to the testnet is enough for this tutorial so we give it a value of https://sdk-testnet.aepps.com, then we create a memory account which uses the secret key and public key our function receives to do this. Then we further create our `nodeInstance` using the `NODE` constructor of the global Ae object. Lastly, we create an SDK instance using the `Universal` constructor on the global `Ae` object and return it. Let's proceed by finally adding the function that will allow us to check our Aeternity wallet account balance.
+
+```javascript
+  async function getAccountBalance(publicKey,sdkInstance){
+    let height=await sdkInstance.height();
+    try{
+        let balance=await sdkInstance.balance(publicKey,{height:+height,hash:null});
```

Comment:
`getBalance` defaults to 0 if account doesn't exist.

no need to pass `height, hash` if not used or default values

Reply:
Noted, i corrected that in the showcase and the tutorial video already, I forgot to correct it here

Reply:
Have corrected this now